types/fuzzyset/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for fuzzset 1.0
+// Project: https://github.com/axiak/fuzzyset
```

Comment:
That page claims to be published as `fuzzyset.js`, which would mean the directory here should be renamed. But that package hasn't been published since 2012.